### ISSUE TITLE
[Bug Report] Vuetify form components not working with v-model.trim as expected

### ISSUE THREAD

**Issue by jakeoverall** [vuetifyjs/vuetify#11375]
### Environment
**Vuetify Version:** 2.2.27
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.138
**OS:** Windows 10

### Steps to reproduce
create any vuetify text input element with v-model bound with the .trim attribute

&lt;v-text-field v-model.trim="x"&gt;
or
&lt;v-textarea v-model.trim="x"&gt;

vs 

&lt;input v-model.trim="x"&gt;
&lt;textarea v-model.trim="x"&gt;

### Expected Behavior
When removing the last letter of a word the trim should not remove the whitespace as vue is smart enough not to trigger the trim until blur

### Actual Behavior
Vuetify components trigger the trim attribute prematurely removing the whitespace causing frustration to users.

### Reproduction Link
<a href="https://codepen.io/jakeoverall/pen/OJyEgwQ?editable=true&editors=101%3Dhttps%3A%2F%2Fvuetifyjs.com%2Fen%2Fcomponents%2Ftextarea%2F" target="_blank">https://codepen.io/jakeoverall/pen/OJyEgwQ?editable=true&editors=101%3Dhttps%3A%2F%2Fvuetifyjs.com%2Fen%2Fcomponents%2Ftextarea%2F</a>

**Comment by dhstack** [comment]
My users are having a similar error, but in this case their cursor jumps to end of an input field. This typically happens on iOS devices as far as I can tell. I'm fairly certain it's related to `v-model.trim`.

Confirmed on:
- iPhone Mobile Safari 13.1.2, iOS 13.6.1.

This is the closest thing I've found relating to this issue: https://stackoverflow.com/questions/55478646/input-cursor-jumps-to-end-of-input-field-on-input-event
